libraries/ui/src/Nav.tsx

```diff
@@ -14,94 +13,152 @@ export type NavProps = React.PropsWithChildren<{
   }>;
 }>;
 
-type NavButtonProps = LinkOrButtonProps;
+const SearchIcon: React.FC = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48" fill="none">
+    <circle cx="21.4853" cy="22.4853" r="6" transform="rotate(-45 21.4853 22.4853)" stroke="currentColor" strokeWidth="2" strokeLinecap="round" />
+    <path d="M27.1421 28.1421L30.6776 31.6776" stroke="currentColor" strokeWidth="2" strokeLinecap="round" />
+  </svg>
+);
+
+const DropdownIcon: React.FC<{ expanded: boolean }> = ({ expanded }) => (
+  <svg
+    className={clsx('size-4 transition-transform', expanded ? 'rotate-180' : '')}
+    viewBox="0 0 24 24"
+    fill="none"
+    stroke="currentColor"
+  >
+    <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M19 9l-7 7-7-7" />
+  </svg>
+);
```

Comment:
Are you considering moving these to the `/icons` folder as well?

Reply:
Yes, I'll do this in a followup PR though to try and clean up a lot of the svg stuff in one go